docker-compose.yml

```diff
@@ -47,23 +48,24 @@ services:
       - "-c"
       - "/app/zkevm-node run --network custom --custom-network-file /app/genesis.json --cfg /app/config.toml --components rpc"
 
-  zkevm-aggregator:
-    container_name: zkevm-aggregator
-    image: 0xpolygon/cdk-validium-node:0.6.2-cdk
+  cdk-validium-aggregator:
+    container_name: cdk-validium-aggregator
+    image: 0xpolygon/cdk-validium-node:0.6.4-cdk
     ports:
       - 50081:50081
       - 9093:9091 # needed if metrics enabled
     volumes:
+      - ./config/node/sequencer.keystore:/pk/sequencer.keystore
```

Comment:
so the new aggregator component need this? what will happen w/o it?

Reply:
from the [aggregator code](https://github.com/0xPolygon/cdk-validium-node/blob/v0.6.4%2Bcdk/aggregator/aggregator.go#L372), it only is used for agg layer.

Reply:
discussed offline. not needed